### PR TITLE
docs: updated inconsistent `@fuel-ts/account` README

### LINKED ISSUE #2193: Inconsistencies with @fuel-ts/account README
As part of my effort to clean up NPM (#1896), I noticed the following inconsistencies with `@fuel-ts/account`. We should update the [README.md](https://github.com/FuelLabs/fuels-ts/blob/master/packages/account/README.md) to better reflect the current package.

### Inconsistencies

- `@fuel-ts/wallet` is incorrect, should be `@fuel-ts/account`
- Inconsistent headings.
- `@account/test-utils` should be `@fuel-ts/account/test-utils`.
- `BaseAssetId` should be `baseAssetId`.

### PR DESCRIPTION
**Pull request by petertonysmith94**
Closes #2193

### CODE REVIEW

File: .changeset/clever-years-invent.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+docs: updated inconsistent @fuel-ts/account README
```
Comment (maschad) [inline .changeset/clever-years-invent.md:5]: I updated the PR 👁️ 

```suggestion
docs: updated inconsistent `@fuel-ts/account` README
```

File: packages/account/README.md [hunk 2]
```diff
@@ -38,13 +38,16 @@ pnpm add fuels
 npm add fuels
 ```
 
-# Test Utilities
+## Test Utilities
```
Comment (maschad) [inline packages/account/README.md:41]: Just curious but why the change to h2?

**Review by maschad — COMMENTED** [review]
Looks good, minor suggestions

File: packages/account/README.md (hunk 2 above)
Comment (maschad) [reply packages/account/README.md:41]: That's reasonable. Good change. I like that you referenced the HTML standard as well.

**Review by maschad — APPROVED** [review]

**Review by arboleya — APPROVED** [review]
Good catch! 🎯
